packages/commonwealth/client/scripts/views/components/sidebar/index.tsx

```diff
@@ -40,39 +64,47 @@ export const Sidebar = ({ isInsideCommunity }) => {
     <div className="Sidebar">
       <div className="sidebar-default-menu">
         <SidebarQuickSwitcher />
-        {app.activeChainId() && isInsideCommunity && (
+        {isInsideCommunity && (
           <div className="community-menu">
-            {featureFlags.sessionKeys && (
-              <AccountConnectionIndicator connected={true} />
-            )}
-            {showAdmin && <AdminSection />}
-            {featureFlags.communityHomepage && app.chain?.meta.hasHomepage && (
-              <div
-                className={onHomeRoute ? 'home-button active' : 'home-button'}
-                onClick={() => navigate('/feed')}
-              >
-                <CWIcon iconName="home" iconSize="small" />
-                <CWText>Home</CWText>
-              </div>
-            )}
-            <DiscussionSection />
-            <GovernanceSection />
-            <ExternalLinksModule />
-            <div className="buttons-container">
-              {isLoggedIn && app.chain && (
-                <div className="subscription-button">
-                  <SubscriptionButton />
+            {!app.activeChainId() ? (
```

Comment:
I would really like us to have a more general / abstract pattern for skeleton loading than an inline conditional. Curious what @masvelio thinks about possible solutions.

Reply:
Yeah, this is problematic, because each component has its own skeleton component. In this example we have Sidebar component div, but also inside it has `sidebar-default-menu`, `community-menu` and  SidebarQuickSwitcher component. Ideally, everywhere where we want to have a skeleton, it should be a separate component so we would be able for example follow the early return patter.

```
if (loading) {
  return <SidebarSkeleton>
}

return <Sidebar>
```

We could also do some wrapper component as an abstraction but this is HoC approach which I do not like too much (it is not modern solution anymore). 

```
return <SkeletonWrapper 
               loading={isLoading} 
               skeletonComponent={SidebarSkeleton} >
                 <Sidebar/>
      </SkeletonWrapper>
```

Reply:
I'm happy with the early return pattern, seems natural to me. cc @mzparacha

Reply:
hm, that seems better. But there would be cases where we only want to show skeleton in a portion of the full component while retaining the functionality (like clicks) of the other parts. Suggestions on those cases? @jnaviask @masvelio.

This component we just skeleton loaded is an example of this. In the skeleton state we can still use the left sidebar to open the explore menu. A way around this would be to divide component logic into several components.

Reply:
Thats what I had in mind. Basically the rule should be simple, every time you want to use skeleton component, it should belong to the individual component, so we can use early return pattern. Otherwise, inline conditional is unavoidable

Reply:
cc @mzparacha to please update the skeleton loaders to use this rule, then we can ship it

Reply:
sure, i will be doing that shortly today

Reply:
Done